### ISSUE TITLE
Create REI Bug Fix does not Support Create 0.5.1b

### ISSUE THREAD

**Issue by suerion** [TexTrueStudio/CreateFabricRecipeREICompatibility#2]
Hey we got some errors on Create and Create ReiBugfix

```
[14:44:50] [main/INFO]: Fix: add [], remove [], replace [[createreibugfix 0.1.4-mc1.19.x] -> add:createreibugfix 1 ([(-Ôê×,Ôê×)])]
[14:44:50] [main/ERROR]: Incompatible mod set!
net.fabricmc.loader.impl.FormattedException: Mod resolution encountered an incompatible mod set!
A potential solution has been determined:
         - Replace mod 'CreateFabric&REIBugFix' (createreibugfix) 0.1.4-mc1.19.x with any version that is compatible with:
                 - create 0.5.1-b-build.1089+mc1.19.2
Unmet dependency listing:
         - Mod 'CreateFabric&REIBugFix' (createreibugfix) 0.1.4-mc1.19.x requires version 0.5.1 of mod 'Create' (create), but only the wrong version is present: 0.5.1-b-build.1089+mc1.19.2!
```

**Comment by suerion** [comment]
https://mclo.gs/Xjmi57s

**Comment by TexBlock** [comment]
will fix in `0.1.5-create0.5.1-mc1.19.x`
